### DISCUSSION TITLE
Welcome to docs discussions!

### DISCUSSION THREAD

**Discussion by dpolivaev** [freeplane/docs#14] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our documentation writers. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kenfhill84083** [comment]
I hate to admit it, but I am overwhelmed by the structure of the Freeplane documentation project.

I've missed a few steps in real time during my hiatus from Freeplane.  Hopefully that accounts for part of the "Alice in Wonderland" sensation I'm feeling.

I am very confused about where and how to jump in.

Is there an index or table of contents?

Is there a "Start Here" or "How To Find Your Place in the Documentation Project" resource?

Is there a logical sequence of posts that will fill me in on how to proceed so I'm on the same page and do not stray from my lane (whatever that is)?

I have been publishing a Freeplane tutorial video each day on Youtube for about a week.  But I realize it is hard to plug in something like that into an overall documentation effort.

**Reply by dpolivaev** [reply]
Creating structure from chaos is hard. Unfortunately, nobody has taken the lead.
